requirements.txt

```diff
@@ -1,5 +1,27 @@
-pika==0.10.0
-requests==2.12.4
-six==1.10.0
-structlog==16.1.0
-sdx-common==0.7.0
+pika==0.10.0 \
```

Comment:
Need to update changelog (see comment on [sdx-rabbit-monitor](https://github.com/ONSdigital/sdx-rabbit-monitor/pull/29/files/c5002976f8e11a3b83c47a3304ce88dfecf8421f) )